client/src/app/components/Pagination.tsx

```diff
@@ -84,11 +131,23 @@ class Pagination extends Component<PaginationProps, PaginationState> {
                   disabled: this.props.currentPage === this.state.lastPage
                 })}
                     onClick={() => {
-                        this.onNext();
+                        this.props.onPageChange(this.props.currentPage + 1);
                     }}
                 >
                     <div className="arrow right" />
                 </li>
+                <li
+                    className={classNames("pagination-item", {
+                  disabled: this.props.currentPage > this.state.lastPage - 10,
+                  "hidden": this.state.isMobile
```

Comment:
Same here about the consistent definition of class names (with or without quote), also formatting.